ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -78,6 +79,12 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
       ? merge({}, DEFAULT_LEGEND, props.spec.legend)
       : undefined;
 
+  // control whether query is visible in TimeSeriesTooltip SeriesInfo component
+  const tooltipConfig: TooltipConfig = {
+    showQuery: props.spec.tooltip?.show_query ?? true,
```

Comment:
I'm thinking the ideal behavior is to automatically set `show_query` to false when there is only 1 query, but set it to true when there is more than 1 (assuming series_name_format wasn't set since that overrides the series name anyways and will still work)